internal/v1/handler_blueprints.go

```diff
@@ -161,10 +161,19 @@ func (h *Handlers) GetBlueprints(ctx echo.Context, params GetBlueprintsParams) e
 	if params.Offset != nil {
 		offset = *params.Offset
 	}
+	var blueprints []db.BlueprintWithNoBody
+	var count int
 
-	blueprints, count, err := h.server.db.GetBlueprints(idHeader.Identity.OrgID, idHeader.Identity.AccountNumber, limit, offset)
-	if err != nil {
-		return err
+	if params.Search != nil {
```

Comment:
I would probably also add a case when `params.Search != ""` because in that case we do a weird query `ILIKE %%%%` and it can easily happen that our JS code will be sending an empty string instead of `nil` every time, thus every single time when the app would request all blueprints we would actually perform an ILIKE search.